DependencyInjection/Configuration.php

```diff
@@ -43,18 +43,29 @@ public function getConfigTreeBuilder()
 
                 ->arrayNode('persistence')
                     ->addDefaultsIfNotSet()
+                    ->beforeNormalization()
+                        ->ifTrue(function ($v) {
+                            return !isset($v['service']) && !isset($v['class']);
+                        })
+                        ->then(function ($v) {
+                            $v['class'] = 'Liip\TranslationBundle\Persistence\YamlFilePersistence';
+                            $v['options']['folder'] = '%kernel.root_dir%/data/translations';
```

Comment:
But how to check if the user configured the service? Then you have to make the assumption that if the service is set, you need to use the service otherwise fall back to the (default) class.